### ISSUE TITLE
Recent commit broke 'make frameworks'

### ISSUE THREAD

**Issue by aepryus** [google/j2objc#1871]
This commit: 3596e501fc92946bc9092eb2637b4fb21e354b2a

broke `make frameworks`

with: `export J2OBJC_ARCHS="iphone64 simulator simulator64 macosx macosx64"`

Here is the error:
```
Both 'macos-arm64' and 'macos-x86_64' represent two equivalent library definitions.
make[1]: *** [../dist/frameworks/JRE.xcframework] Error 70
make: *** [jre_emul_dist] Error 2
```

**Comment by dsanghan** [comment]
+1

**Comment by tomball** [comment]
Fixed with commit [668ed8f](https://github.com/google/j2objc/commit/668ed8f7c0693fb55ac05c75050977f148491526).
